REDHOUSEVE/edx-platform
# Review Discussion Forum Turned Off

**Description:** 

**JIRA:** 
https://edlyio.atlassian.net/browse/EDE-943

**Testing instructions:**

1. Open LMS dashboard
2. Click on a course
3. The discussion tab should not be visible

<img width="507" alt="Screenshot 2020-09-16 at 3 52 19 PM" src="https://user-images.githubusercontent.com/29246328/93328040-9e892f80-f834-11ea-8e4f-95840af0acbe.png">
